src/dmg2john.c

```diff
@@ -428,10 +490,27 @@ static void hash_plugin_parse_hash(char *in_filepath)
 		free(chunk1);
 		free(v2_password_header.keyblob);
 	}
+
+bailout:
 	close(fd);
 }
 
-int dmg2john(int argc, char **argv)
+#ifdef HAVE_LIBFUZZER
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
+{
+	int fd;
+	char name[] = "/tmp/libFuzzer-XXXXXX";
+
+	fd = mkstemp(name);
```

Comment:
Thanks! I have added error handling now.

Such a check would be useful for detecting problems with the setup of test cases.